docs/deep-dive/03-dd-expose-function-serverless.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Expose a Function with an API Rule
-type: Tutorials
+type: Deep Dive
 ---
 
 This tutorial shows how you can expose your Function to access it outside the cluster, through an HTTP proxy. To expose it, use an APIRule custom resource (CR) managed by the in-house API Gateway Controller. This controller reacts to an instance of the APIRule CR and, based on its details, it creates an Istio Virtual Service and Oathkeeper Access Rules that specify your permissions for the exposed Function.
```

Comment:
```suggestion
This tutorial shows how you can expose your Function to access it outside the cluster through an HTTP proxy. To expose it, use an APIRule custom resource (CR) managed by the in-house API Gateway Controller. This controller reacts to an instance of the APIRule CR and, based on its details, it creates Istio Virtual Service and Oathkeeper Access Rules that specify your permissions for the exposed Function.
```

Reply:
Actually, that is ok as I mean separately `an Istio Virtual Service` and `Oathkeeper Access Rules`